integration-tests/src/test/java/oracle/weblogic/kubernetes/ItManageNs.java

```diff
@@ -0,0 +1,824 @@
+    createSecretWithUsernamePassword(adminSecretName, domainNamespace, "weblogic", "welcome1");
+
+    // create encryption secret
+    logger.info("Creating encryption secret");
+    createSecretWithUsernamePassword(encryptionSecretName, domainNamespace, "weblogicenc", "weblogicenc");
+  }
+
+  private static void deleteSecrets(String domainNamespace) {
+    logger.info("Deleting docker registry secret in namespace {0}", domainNamespace);
+    if (!domainNamespace.equals("default")) {
+      deleteSecret(OCR_SECRET_NAME, domainNamespace);
+    }
+
+    // delete secret for admin credentials
+    logger.info("Deleting secret for admin credentials");
+    deleteSecret(adminSecretName, domainNamespace);
+
+    // delete encryption secret
+    logger.info("Deleting encryption secret");
+    deleteSecret(encryptionSecretName, domainNamespace);
+  }
+
+  private void createVerifyDomain(String domainNamespace, String domainUid, String miiImage, Domain domain) {
+    // create domain
+    logger.info("Creating model in image domain {0} in namespace {1} using docker image {2}",
+        domainUid, domainNamespace, miiImage);
+    createDomainAndVerify(domain, domainNamespace);
+    String adminServerPodName = domainUid + adminServerPrefix;
+    // check that admin server pod exists in the domain namespace
+    logger.info("Checking that admin server pod {0} exists in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodReadyAndServiceExists(adminServerPodName, domainUid, domainNamespace);
+
+    // check for managed server pods existence in the domain namespace
+    for (int i = 1; i <= replicaCount; i++) {
+      String managedServerPodName = domainUid + managedServerPrefix + i;
+
+      // check that the managed server pod exists
+      logger.info("Checking that managed server pod {0} exists in namespace {1}",
+          managedServerPodName, domainNamespace);
+      checkPodReadyAndServiceExists(managedServerPodName, domainUid, domainNamespace);
+    }
+  }
+
+  /**
+   * Install WebLogic operator and wait up to two minutes until the operator pod is ready.
+   *
+   * @param operNamespace the operator namespace in which the operator will be installed
+   * @param opReleaseName the operator release name
+   * @param enableClusterRoleBinding operator cluster role binding
+   * @param domainNamespace the list of the domain namespaces which will be managed by the operator
+   * @return the operator Helm installation parameters
+   */
+  private static HelmParams installOperatorHelmChart(String opReleaseName, String operNamespace,
+                                                     String domainNsSelectionStrategy,
+                                                     String domainNsSelector,
+                                                     boolean enableClusterRoleBinding,
+                                                     String... domainNamespace) {
+    LoggingFacade logger = getLogger();
```

Comment:
You can define a class level logger and use it in all the methods

Reply:
done